packages/flutter/test/widgets/semantics_tester.dart

```diff
@@ -48,6 +48,8 @@ class TestSemantics {
     this.transform,
     this.textSelection,
     this.children = const <TestSemantics>[],
+    this.scrollIndex,
+    this.scrollChildren,
```

Comment:
One idea I had in the past, but never got to implement, was to make this more flexible by taking `Matcher` as a value. Then you could use an `any` matcher to indicate that you don't care about the value. I don't know if it's helpful in this case.